atomate/qchem/powerups.py

```diff
@@ -2,15 +2,22 @@
 
 
 from atomate.qchem.firetasks.run_calc import RunQChemFake
+from atomate.qchem.firetasks.critic2 import RunCritic2
 
-__author__ = "Brandon Wood"
-__email__ = "b.wood@berkeley.edu"
+__author__ = "Brandon Wood", "Samuel Blau"
+__copyright__ = "Copyright 2019, The Materials Project"
+__version__ = "0.1"
+__maintainer__ = "Samuel Blau"
+__email__ = "samblau1@gmail.com"
+__status__ = "Alpha"
+__date__ = "11/20/19"
 
 
 def use_fake_qchem(original_wf, ref_dirs, input_file="mol.qin"):
     """
         Replaces all RunQChem commands (i.e. RunQChemDirect, RunQChemCustodian) with RunQChemFake.
         This allows for testing without actually running QChem
+        Also turns on testing=True for RunCritic2 to similarly facilitate testing
```

Comment:
Instead implement a RunFakeCritic2 or even just delete the `RunCritic2` firetask.

Reply:
FYI, I went with the second option and now just delete the `RunCritic2` as part of `use_fake_qchem`. It feels a little weird to do something critic-specific in what appears to be a qchem-specific function, but since Critic2 is run in the same directory as the QChem calculation, and thus the Critic2 outputs are "generated" via emulation along with the QChem outputs, this is definitely the simplest solution.